src/Polly.Strategies/Timeout/Internals/TimeoutResilienceStrategy.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.Extensions.ObjectPool;
+using Polly.Internals;
+
+namespace Polly.Timeout.Internals;
+
+internal sealed class TimeoutStrategy : DelegatingResilienceStrategy
+{
+    private static readonly ObjectPool<CancellationTokenSource> _cancellations = ObjectPool.Create<CancellationTokenSource>();
```

Comment:
In another class this is prefixed with `s_`. I'd prefer without, but whichever is chosen it would be good to be consistent.